packages/sil-voting-app/src/routes/dashboard/utils/randomProposal.ts

```diff
@@ -0,0 +1,96 @@
+import { ProposalProps, VoteResult } from '../components/Proposal';
```

Comment:
Where do you use those utilities? I think you should place them inside: `dashboard/utils/test/randomProposal.ts`